openresty/Dockerfile-alpine

```diff
@@ -0,0 +1,254 @@
+# Current latest Openresty version is here.
+# Also specify the nginx core used. (Openresty, adds their own version numbers after the nginx one, so we can't use the same number for both Openresty and nginx)
+# Also specify the LUA version used. (Comes with the latest Openresty src bundle.)
+ARG OPENRESTY_VERSION="1.21.4.3" \
```

Comment:
```suggestion
ARG OPENRESTY_VERSION="1.25.3.1" \
```

Reply:
Will change. Finally openresty has been updated to a newer version of nginx.

Reply:
Sorry, I didn't mean to put additional work on your plate, I simply wanted the versions to be consistent across all images.

Reply:
No worries